e2e-tests/utils/Login.ts

```diff
@@ -33,10 +35,19 @@ export class Login {
 
   private static waitForApps() {
     Common.waitForLoad();
+    // Wait for https://localhost:8080/ loaded
     GetStartedPage.waitForLoad();
-    Common.navigateTo(NavItem.applications);
+    // Go to the https://localhost:8080/workspaces
+    Common.navigateTo(NavItem.namespaces);
+    Common.verifyPageTitle(pageTitles.namespaces);
+    Common.waitForLoad();
+    cy.testA11y(`${pageTitles.namespaces}`);
+    // Go to https://localhost:8080/workspaces/your-tenant/applications
+    cy.get(goToApplicationsPagePo.goToApplicationsPagePo).first().click();
```

Comment:
I think it should be more stable if we search by `HAC_NAMESPACE` or `HAC_WORKSPACE` variable instead of grabbing the first namespace from the list. If it should be the first one in the list, we may check it before clicking, something like:

Reply:
+1, let me change this part.